aldryn_events/tests/frontend/integration/pages/page.events.crud.js

```diff
@@ -0,0 +1,66 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global by, element, browser, expect */
+
+// #############################################################################
+// INTEGRATION TEST PAGE OBJECT
+
+var eventsPage = {
+    site: 'http://127.0.0.1:8000/en/',
```

Comment:
is this required here @vxsx @Marketionist as the base url is added here: https://github.com/aldryn/aldryn-events/pull/109/files#diff-b9e12334e9eafd8341a6107dd98510c9R104